Codeinwp/neve
# Separator not visible inside the editor

### Summary
- The border was overwritten from the core. I added `!important` as it is more efficient than making the selector more specific. The instruction that overwrites our code looks like this: `.block-editor-block-list__block[data-type="core/separator"].wp-block-separator.is-style-dots`

### Will affect visual aspect of the product
NO


### Test instructions
- Create a post and add a block separator.
- Set the style of that separator to dotted.
- Check if the separator appears in both editor and frontend

## Check before Pull Request is ready:




Closes #3888.
